### PR TITLE
Registers package with the sunpy packaging

### PR DESCRIPTION
**Pull request by CyclingNinja**

### CODE REVIEW

File: radiospectra/net/sources/tests/test_ecallisto_client.py
Comment (samaloney) [inline radiospectra/net/sources/tests/test_ecallisto_client.py:1]: I guess need to add 'technik' to the spell check list of words

File: radiospectra/net/sources/tests/data/rstn_learmonth.html
Comment (samaloney) [inline radiospectra/net/sources/tests/data/rstn_learmonth.html:1]: Should probably skip anything update `/**/data/` or anything not a .py file?

Comment (CyclingNinja) [reply radiospectra/net/sources/tests/data/rstn_learmonth.html:1]: yeah excellent call, i'm assuming this is broken for a test?

File: radiospectra/net/sources/tests/test_ecallisto_client.py
Comment (CyclingNinja) [reply radiospectra/net/sources/tests/test_ecallisto_client.py:1]: I can do that here, dw

**Comment by CyclingNinja** [comment]
@samaloney the error on the py312 build related to the polarization check on the ilofar search, i think its as simple as
```python
        if len(pol) == 1:
            pol = pol[0].upper()
            mask = mask & query_response["Polarisation"] == pol
```
but I wanted to check

**Comment by samaloney** [comment]
Yea that should work 👍  I'm now questioning if that code even makes sense but that's another issue altogether ☹️

**Review by nabobalis — APPROVED** [review]

**Pull request merged**